rg3d-ui/src/formatted_text.rs

```diff
@@ -498,6 +510,16 @@ impl FormattedTextBuilder {
         self
     }
 
+    pub fn with_mask_char(mut self, mask_char: char) -> Self {
```

Comment:
I think one method that accepts `Option<char>` should be enough. `with_no_mask_char` is a bit confusing because you have no mask char by default.